illwill.nim

```diff
@@ -455,6 +649,11 @@ else:  # OS X & Linux
                                   # we'll ignore them
       else:
         key = toKey(ch)
+
+    elif charsRead > 3 and keyBuf[0] == 27 and keyBuf[1] == 91 and keyBuf[2] == 60:
```

Comment:
Consider extracting these magic numbers into constants with meaningful names to aid readability. In case of printable ASCII chars you could just do `ord('X')`.